data/remnant missions.txt

```diff
@@ -60,27 +62,94 @@ mission "First Contact: Remnant"
 	
 	on decline
 		event "ember waste label"
+		set "Remnant Blood Test Pure"
 		log "Discovered a strange, isolated human community in an otherwise uninhabited region of space south of the Core. They call themselves the Remnant."
 		log "Factions" "Remnant" `Half a millennium ago, during the Alpha Wars, a group of humans fleeing the conflict and looking for a safer place to live discovered a wormhole into a region of space known as the Ember Waste. They built settlements there, and chose to call themselves the Remnant. Although they no longer fear the Alphas as much as they once did, they still have not rejoined human society.`
 			`Since their colonies were first formed, the Remnant's culture has changed drastically. They communicate in sign language, using their voices only for singing. Their food and architecture and much of their technology is strange, and much of it has been borrowed or stolen from nearby alien species.`
 	
 	npc kill
 		government "Remnant"
 		personality heroic coward disables plunders
-		fleet "Large Remnant" 2
+		fleet "Large Remnant" 1
 
 event "ember waste label"
 	galaxy "label waste"
 		sprite label/waste
+		
+mission "Remnant: Second Contact"
+	name "Second chance at First Contact with the Remnant"
+	landing
+	invisible
+	source
+		government "Remnant"
+	description "An A Remnant provides you with the opportunity to take a second run at first contact."
```

Comment:
Remove "An". And why this is invisible?

Reply:
Then if this is invisible, then a name and description is not needed, since it won't be seen anyway. But it is up to you if you will remove them, since the result is the same.

Reply:
I think I'll leave it for now, since I don't see any problem with having it there. (and if something does go wrong - a bug or something that makes it visible - then people will have some info as to what it is) Also, given how many frequently "check your save file" is a response to dealing with mission problems, I think it might be helpful for there to be a bit more consistency in having the "visible" names & short descriptions be a standard thing to include even when they might not be actively shown by the game.